client/app/styles/main.less

```diff
@@ -958,6 +947,25 @@ h2 {
   }
 }
 
+.node-control-button {
+  .btn-opacity;
+  padding: 6px;
+  margin-left: 2px;
+  font-size: 110%;
```

Comment:
Yes, let's aim for `em`. (This needs a proper cleanup PR, there is a `body {font-size: 13px}` lurking.)